doc-content/drools-docs/src/main/asciidoc/DMN/dmn-properties-ref.adoc

```diff
@@ -52,14 +52,26 @@ When valorized with a Java fully qualified name, this property loads a DMN profi
 org.kie.dmn.runtime.listeners.$LISTENER_NAME::
 When valorized with a Java fully qualified name, this property loads and registers a DMN Runtime Listener onto the {DECISION_ENGINE} at start time.
 You can use this property to register a DMN listener in order to be notified of several events during DMN model evaluations.
-You can also configure this property in the `kmodule.xml` file in your project.
 +
 --
 ----
 -Dorg.kie.dmn.runtime.listeners.mylistener=org.acme.MyDMNListener
 ----
 //kept removed `[source]` for this last snippet because it rendered unlike all the others in community output otherwise for some reason. as per SJR comment above.
 --
++
+You can also configure this property in the `kmodule.xml` file in your project.
++
+--
+----
+<kmodule xmlns="http://www.drools.org/xsd/kmodule">
+  <configuration>
+    <property key="org.kie.dmn.runtime.listeners.mylistener" value="org.acme.MyDMNListener"/>
+  </configuration>
+</kmodule>
+----
+//kept removed `[source]` for this last snippet because it rendered unlike all the others in community output otherwise for some reason. as per SJR comment above.
```

Comment:
@jomarko , @hmanwani-rh , I had tried that before, a community rendering issue nonetheless. However, since then a several pom updates occurred, so I just retried, and it's apparently resolved.

So @hmanwani-rh , you can remove the comments and add the source definitions. Also, beware of the formatting use of `+` and `--` You use them to start and end the block, not between every line as you have done.

For you convenience, you can simply copy and paste the following segment entirely over the same in your file. This contains the updated source defs, removed comments, your property update, and the correct formatting.

```
org.kie.dmn.profiles.$PROFILE_NAME::
When valorized with a Java fully qualified name, this property loads a DMN profile onto the {DECISION_ENGINE} at start time. You can use this property to implement a predefined DMN profile with supported features different from or beyond the DMN standard. For example, if you are creating DMN models using the Signavio DMN modeller, use this property to implement features from the Signavio DMN profile into your DMN decision service.
+
--
[source]
----
-Dorg.kie.dmn.profiles.signavio=org.kie.dmn.signavio.KieDMNSignavioProfile
----
--

org.kie.dmn.runtime.listeners.$LISTENER_NAME::
When valorized with a Java fully qualified name, this property loads and registers a DMN Runtime Listener onto the {DECISION_ENGINE} at start time. You can use this property to register a DMN listener in order to be notified of several events during DMN model evaluations.
+
--
[source]
----
-Dorg.kie.dmn.runtime.listeners.mylistener=org.acme.MyDMNListener
----

You can also configure this property in the `kmodule.xml` file in your project.

[source,xml]
----
<kmodule xmlns="http://www.drools.org/xsd/kmodule">
  <configuration>
    <property key="org.kie.dmn.runtime.listeners.mylistener" value="org.acme.MyDMNListener"/>
  </configuration>
</kmodule>
----
--

org.kie.dmn.compiler.execmodel::
When enabled, this property enables DMN decision table logic to be compiled into executable rule models during run time. You can use this property to evaluate DMN decision table logic more efficiently. This property is helpful when the executable model compilation was not originally performed during project compile time. Enabling this property may result in added compile time during the first evaluation by the {DECISION_ENGINE}, but subsequent compilations are more efficient.
+
--
Default value: `false`

[source]
----
-Dorg.kie.dmn.compiler.execmodel=true
----
--
```

Reply:
@sterobin, thank you, do you wish my approval or should I recheck one those source blocks are updated? For me both is fine.

Reply:
@sterobin Thanks, will make the suggested changes.

Reply:
@jomarko, up to @hmanwani-rh , but imo, not necessary for your re-approval.

Reply:
ok, please assume my approval as final